### PR TITLE
[MS-644] Add continue anyway to the ValidateSubjectPoolFragment

### PR DESCRIPTION
**Pull request by meladRaouf**

### CODE REVIEW

File: feature/validate-subject-pool/src/main/res/layout/fragment_validate_subject_pool.xml [hunk 1]
```diff
@@ -71,7 +71,20 @@
...
+
+
+        <com.google.android.material.button.MaterialButton
+            android:id="@+id/validationActionsContinue"
+            style="@style/Widget.Simprints.Button.TextButton.White"
+            android:layout_width="0dp"
+            android:layout_height="96dp"
+            android:layout_marginTop="2dp"
+            android:text="@string/id_pool_validation_action_continue"
+
```
Comment (luhmirin-s) [inline feature/validate-subject-pool/src/main/res/layout/fragment_validate_subject_pool.xml:84]: Nit-pick: couple of random unnecessary empty lines.

File: infra/resources/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -438,6 +438,7 @@
     <string name="id_pool_validation_pool_empty_message">There are no records on the device</string>
     <string name="id_pool_validation_action_close">Close</string>
     <string name="id_pool_validation_action_sync">Sync and retry</string>
+    <string name="id_pool_validation_action_continue">Ignore and Continue</string>
```
Comment (luhmirin-s) [inline infra/resources/src/main/res/values/strings.xml:441]: Is this the final text or a placeholder?

**Review by luhmirin-s — APPROVED** [review]

File: infra/resources/src/main/res/values/strings.xml (hunk 2 above)
Comment (meladRaouf) [reply infra/resources/src/main/res/values/strings.xml]: @luhmirin-s I have added the final text

File: feature/validate-subject-pool/src/main/res/layout/fragment_validate_subject_pool.xml (hunk 1 above)
Comment (meladRaouf) [reply feature/validate-subject-pool/src/main/res/layout/fragment_validate_subject_pool.xml:82]: done

**Review by luhmirin-s — APPROVED** [review]

**Review by alexandr-simprints — APPROVED** [review]

**Pull request merged**